classes/checkout/CheckoutSession.php

```diff
@@ -196,4 +196,16 @@ public function getCheckoutURL()
     {
         return $this->context->link->getPageLink('order');
     }
+
+    public function setSeparatePackagesAllowed(bool $separatePackagesAllowed): bool
+    {
+        $this->context->cart->allow_separated_package = $separatePackagesAllowed;
```

Comment:
@thalassa-web This one is the bad property :

```suggestion
        $this->context->cart->allow_seperated_package = $separatePackagesAllowed;
```

Reply:
You are right … I missread this …
I thought I used the good property but I did not …
Thanks